### ISSUE TITLE
[question] What heppened with the CMakeLists.txt file in the repository root?

### ISSUE THREAD

**Issue by yurivict** [Dav1dde/glad#454]
We have the FreeBSD port graphics/glad with version 0.1.36 that builds a cmake-based project.
Now 2.x.x doesn't have a cmake file any more.
Was the project re-written in Python?

**Comment by Dav1dde** [comment]
The CMake files just moved into the `cmake` directory. Usage changed also a bit, e.g. see [one of the examples](https://github.com/Dav1dde/glad/tree/glad2/example/c%2B%2B/multiwin_mx)

**Comment by yurivict** [comment]
cmake/CMakeLists.txt doesn't build anything though.
It doesn't have the ```add_library(glad ...)``` instruction.
No other CMakeLists.txt has such instruction.

**Comment by Dav1dde** [comment]
Correct, glad was never supposed to be built either, it should be integrated directly in tools which use it.

**Comment by manuelkNVDA** [comment]
as is, the CMakeLists build of the included code sample is broken and there doesn't appear to be any instructions on how to get this code sample to work

```
glad\example\c++\multiwin_mx\multiwin_mx.cpp(3,10): error C1083: Cannot open include file: 'glad/gl.h': No such file or director
```

i would also suggest having cmake fetch glfw automatically if it is not yet a target.
